### DISCUSSION TITLE
How will the winners of validator wave3 be picked

### DISCUSSION THREAD

**Discussion by vickman787** [MystenLabs/sui#8547] [Q&A]
Will we get any any mail from sui if we get picked for the validator wave 3

**Comment by Roxbic** [comment]
> Will we get any any mail from sui if we get picked for the validator wave 3

well, since they asked for email i believe we'll be getting an email. otherwise all info will be passed on discord and twitter.

**Reply by Ubee-James** [reply]
Absolutely, mail makes sense

**Comment by 0xOJ** [comment]
probably they will check people with little experience running validators nodes on other chain

**Comment by paullo240** [comment]
I guess we might be notified through mail.

**Comment by Revelation01** [comment]
I believe an email will be sent
